meson.build

```diff
@@ -0,0 +1,5 @@
+project('src', 'c',
+  version : run_command(['git', 'describe', '--tags'], capture : true, check : true).stdout().strip(),
```

Comment:
This will completely break when using the uploaded tarballs on the releases page.

A better idea might be either having a release script to update the version number in meson.build, commit and tag it.